app/controllers/workspaces_controller.rb

```diff
@@ -0,0 +1,9 @@
+class WorkspacesController < ApplicationController
+  def index
+    @workspace = Workspace.find(params[:id])
```

Comment:
I think we need to change these around, plural with the index and vice versa (as we are listing all instances in the index view) and only one with the show route.